instrumentation/src/javatest/org/oppia/android/instrumentation/BUILD.bazel

```diff
@@ -0,0 +1,34 @@
+"""
+This library contains the test suite source files of the instrumentation module.
+Note that:
+ - Each source file has its own kt_android_library target
+ - Target name of each kt_android_library as to be in small letters, separated by a underscore
+ and ends with "_lib"
+"""
+
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+# Common deps for each test suite.
```

Comment:
What purpose does this serve vs. directly depending on the needed dependencies? We usually prefer the latter since shared dependency groups can make large-scale migrations more difficult, and potentially overcomplicate the build graph if some tests get extra dependencies that they technically don't need.